tests/utils.py

```diff
@@ -549,3 +549,25 @@ def mocked_aws_s3_bucket_test_file(bucket_name, file_name, file_content="Test fi
         tmp_file.flush()
         s3.upload_file(Bucket=bucket_name, Filename=tmp_file.name, Key=file_name)
     return "s3://{}/{}".format(bucket_name, file_name)
+
+
+def mocked_http_test_file(dirname, file_url, file_content="This is a generated file for http test"):
+    # type: (str,str,str) -> str
+    """
+    Generates a test file reference from dummy data
+    """
+    path = dirname+file_url.split("localhost")[-1]
+    with open(path, "w") as tmp_file:
+        tmp_file.write(file_content)
+    return path
+
+
+def mocked_file_test(dirname, file_path, file_content="This is a generated file for file test"):
+    # type: (str,str,str) -> str
+    """
+    Generates a test file reference from dummy data
+    """
+    path = dirname+"/"+file_path
+    with open(path, "w") as tmp_file:
+        tmp_file.write(file_content)
+    return "file://{}".format(path)
```

Comment:
I don't think there is sufficient operations involved here to be worth having a dedicated function for it. 
The test itself could just dump the file directly where needed. 

Also, avoid using "mocked" wording here, nothing is being mocked. This makes the test definition more confusing.